### PR TITLE
Code Migration

### PR DESCRIPTION
**Pull request by tianjianfeng**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,207 @@
...
+## Running the service locally
+By default, the service runs on 8998 port.
+
+#### Using service manager 2
+
+The following sm2 command starts the service but its also part of `GMS_ALL` profile
+
+`sm2 --start GOODS_MOVEMENT_SYSTEM_PORT_API`
+
+the logs can be viewed by running `sm2 --logs GOODS_MOVEMENT_SYSTEM_PORT_API` in a different terminal window
```
Comment (tipperba) [inline README.md:22]: If we can take out references to service manager, that would be great.

File: README.md [hunk 2]
```diff
@@ -1,8 +1,207 @@
...
+For local development use `sbt run` but if its already running in sm2, we should stop it first before running sbt command
+
+The endpoint in this API is application-restricted.
+
+All ports requiring access to this API must be pre-authorized by HMRC.
+
+service logs can be viewed by running `sm --logs GOODS_MOVEMENT_SYSTEM_PORT_API`
+
+```GET  /:portId/arrivals/controlled```
+
```
Comment (tipperba) [inline README.md:34]: if we can remove this bit as well

File: README.md [hunk 3]
```diff
@@ -1,8 +1,207 @@
...
+Port can pull all inspection required GMR's that are arriving at their port.
```
Comment (tipperba) [inline README.md:35]: If possible, can we change this: 

`Port can pull all inspection required GMR's that are arriving at their port.`

to this: 

Retrieve all GMRs arriving at a specified port which require an inspection."

File: README.md [hunk 4]
```diff
@@ -1,8 +1,207 @@
...
+          "L0029A"
+        ]
+      }
+    ],
+} ]
+```
+
+```GET /:portId/departures/controlled```
+
+Port can pull all inspection required GMR's that are departing from their port.
```
Comment (tipperba) [inline README.md:62]: Similarly, can we change this to:

`Retrieve all GMRs departing from a specified port which require an inspection.`

File: README.md [hunk 5]
```diff
@@ -1,8 +1,207 @@
...
+  "checkedInCrossing" : {
+    "routeId" : "9",
+    "localDateTimeOfArrival" : "2021-08-11T10:58"
+  }
+} ]
+```
 
-This is a placeholder README.md for a new repository
+```GET  /reference-data ``` <br/>
+The Carrier can get a reference data at anytime.
```
Comment (tipperba) [inline README.md:133]: Can we change this: 

`The Carrier can get a reference data at anytime.`

to: 

`Retrieves reference data on request`

File: README.md [hunk 6]
```diff
@@ -1,8 +1,207 @@
...
+    {
+      "ruleId": "014_2",
+      "ruleDescription": "The EORI cannot be found. Check and try again."
+    }
+  ]
+}
 
-This code is open source software licensed under the [Apache 2.0 License]("http://www.apache.org/licenses/LICENSE-2.0.html").
\ No newline at end of file
+```
```
Comment (tipperba) [inline README.md:207]: Do we need any licence information in the README? (what does it say in the guides?)

File: resources/public/api/conf/1.0/examples/get-reference-data-example.json [hunk 7]
```diff
@@ -0,0 +1,67 @@
...
+    {
+      "carrierId": 2,
+      "carrierName": "DFDS Seaways",
+      "countryCode": "DK"
+    }
+  ],
+  "ruleFailures": [
+    {
+      "ruleId": "BR005",
+      "ruleDescription": "A GMR can only be used in one crossing between two customs territiories"
```
Comment (tipperba) [inline resources/public/api/conf/1.0/examples/get-reference-data-example.json:60]: Typo in the word `territiories`

should say "territories"

File: resources/public/api/conf/1.0/schemas/gmr-types-schema.json [hunk 8]
```diff
@@ -0,0 +1,513 @@
...
+          "masterRefNum": { "$ref": "#/definitions/sAndSMasterRefNum" }
+        }
+      }
+    },
+    "transitDeclarations": {
+      "$id": "#/definitions/transitDeclarations",
+      "type": "array",
+      "description": "Details of the transit declarations within this consignment",
+      "items": {
+        "$id": "#/definitions/transitDeclarations/items",
```
Comment (tipperba) [inline resources/public/api/conf/1.0/schemas/gmr-types-schema.json:446]: I think a lot of the stuff in this file isn't needed for the port-api and was probably copied across when the service was first created. Would be good to cut this file down to the essentials before we go public.

**Comment by tipperba** [comment]
checks currently failing

**Comment by Mahesh49** [comment]
Test comment

**Comment by hmrc-web-operations** [comment]
Thank you for submitting a pull request on this HMRC repository. A member of the HMRC team will review your changes and action accordingly.

HMRC reviewer: please reference [this runbook](https://github.com/hmrc/build-and-deploy/blob/main/runbooks/reviewing_external_pull_requests.md) for guidance on how to proceed.

**Pull request closed without merging**